flink-ml-parent/flink-ml/src/main/java/org/apache/flink/ml/params/statistics/SummarizerParams.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.flink.ml.params.statistics;
+
+import org.apache.flink.ml.params.shared.colname.HasSelectedColNamesDvNull;
+import org.apache.flink.ml.params.shared.colname.HasWeightColNameDvNull;
+
+/**
+ * Params of Summarizer.
+ */
+public interface SummarizerParams<T> extends
```

Comment:
I suppose the SummarizerParams is not a common param interface but specific for Summarizer. I think it's better to move this interface to the PR containing Summarizer.

Reply:
Yes, let's discuss this params in the other PR.